library/sbi-spec/src/fwft.rs

```diff
@@ -9,10 +10,12 @@ mod fid {
     /// Set the firmware function of the request based on Value and Flags parameters.
     ///
     /// Declared in §18.1.
+    #[doc(alias = "sbi_set")]
```

Comment:
```suggestion
    #[doc(alias = "sbi_fwft_set")]
```
Add the module name back for C language functions.